source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,95 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the replication option you use for the site.
+
+.. list-table:: 
+   :widths: 25 75
+   :width: 100%
+
+   * - Site Replication
+     - Total restoration from the healthy peer site(s)
+   * - Bucket Replication
+     - Data restoration of objects and metadata from a healthy remote location for each bucket configured for replication
+   * - :mc-cmd:`mc mirror`
+     - Data restoration of objects only from a healthy remote location with no versioning
+
+Site Replication
+----------------
+
+:ref:`Site replication <minio-site-replication-overview>` keeps two or more MinIO deployments in sync with IAM policies, buckets, bucket configurations, objects, and object metadata.
+If a peer site fails, such as due to a major disaster or long power outage, you can use the remaining healthy site(s) restore the :ref:`replicable data <minio-site-replication-what-replicates>`.
+
+The following procedure can restore data in scenarios where :ref:`site replication <minio-site-replication-overview>` was active prior to the site loss.
+
+1. Deploy a new MinIO site using the same ``root`` credentials as used on other deployments in the site replication configuration
+
+   You can use the original hardware, if still available, but you must first wipe any remaining data before creating the new site.
+2. Configure the new site with the same Identity Provider (IDp) as the other sites
+3. :ref:`Expand the existing site replication <minio-expand-site-replication>` by adding the newly deployed site
+4. Remove the lost site from the site replication configuration
+
+Site replication healing automatically adds IAM settings, buckets, bucket configurations, and objects from the existing site(s) to the new site with no further action required.
+
+If you are switching from using bucket replication to using site replication, you must first remove all bucket replication rules from the healthy site prior to setting up site replication.
+
+Replication Resync
```

Comment:
Rename to 'Bucket Replication' to make it findable

Reply:
Updated heading per the suggestion below.
Added additional context about switching from bucket replication to site replication.